travel_platform/Html/main.html

```diff
@@ -1,111 +1,127 @@
 {% extends 'blog/base.html' %}
 {% block content %}
 <header>
-    <div class="row head-bkg">
-      <div class="col-md-offset-3 col-md-6 head-text text-center">YOUR<br>TRAVEL</div>
+  <div class="row head-bkg">
+    <div class="col-md-offset-3 col-md-6 head-text text-center">YOUR<br>TRAVEL
     </div>
-  </header>
+  </div>
+</header>
+
+<div class="row">
+  <div class="col-md-offset-4 col-md-8"><h2 class="text-center white">Common information</h2>
+  </div>
+
+  <div class="col-md-4 choose">
+    <a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/planet.jpg" alt="planet">
+      <span class="text-center">Choose a city</span>
+    </a>
+  </div>
 
-  <div class="row">
-    <div class="col-md-offset-4 col-md-8"><h2 class="text-center white">Common information</h2>
+  <div class="col-md-8">
+    <!-- TODO roman: Fix this, because it's not expected to have 6th `col-md-4`
```

Comment:
@RomanPryima , FYI. I suggest to merge how it is and fix this as you'll have a free time or you'll change something in this file.